src/corehost/cli/args.h

```diff
@@ -42,6 +42,14 @@ struct probe_config_t
     {
     }
 
+    bool is_lookup() const
```

Comment:
Can you please help me understand the rationale behind this and how it maps to "does not have nuget layout"?